pyproject.toml

```diff
@@ -39,11 +39,6 @@ dbt-duckdb = { version = ">=1.0.0", optional = true }
 dbt-sqlite = { version = ">=1.0.0", optional = true }
 dbt-postgres = { version = ">=1.0.0", optional = true }
 
-[tool.poetry.dev-dependencies]
```

Comment:
`dev-dependencies` is deprecated.

- ref: https://python-poetry.org/docs/managing-dependencies/